pandas/tests/frame/methods/test_transpose.py

```diff
@@ -129,7 +129,10 @@ def test_transpose_get_view_dt64tzget_view(self):
         assert result._mgr.nblocks == 1
 
         rtrip = result._mgr.blocks[0].values
-        assert np.shares_memory(arr._ndarray, rtrip._ndarray)
+        if using_copy_on_write:
+            assert not np.shares_memory(arr._ndarray, rtrip._ndarray)
```

Comment:
Similarly here, for the intent of the test, I think we should still try to verify that `df.T` shares the memory with `df`?

Reply:
Done